packages/es-components/src/components/containers/heading/Heading.js

```diff
@@ -3,24 +3,51 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
 const HeadingBase = styled.h1`
-  color: inherit;
-  font-size: ${props => props.theme.headingSize[props.adjustedSize]};
+  background-color: ${props =>
+    props.knockout && props.theme.colors.softwareBlue};
+  border-bottom: ${props =>
+    props.underlineColor && `2px solid ${props.underlineColor};`};
+  color: ${props => (props.knockout ? 'white' : 'inherit')};
+  font-size: ${props =>
+    props.adjustedSize > 2
+      ? props.theme.headingSize[props.adjustedSize]
+      : `calc(${props.theme.headingSize[props.adjustedSize]} - 6px);`};
   font-weight: 300;
   line-height: 1.1;
-  margin-bottom: 12.5px;
-  margin-top: 25px;
+  margin-bottom: 0.45em;
+  margin-top: 0;
+  padding-bottom: ${props => props.underlineColor && '0.22em'};
+  padding: ${props => props.knockout && '20px 15px'};
 
   small {
     font-size: ${props => (props.adjustedSize > 3 ? '75%' : '65%')};
     line-height: 1;
   }
+
+  @media (min-width: ${props => props.theme.screenSize.tablet}) {
+    font-size: ${props => props.theme.headingSize[props.adjustedSize]};
+  }
 `;
 
-function Heading({ children, level, size, ...other }) {
+function Heading({
+  children,
+  level,
+  size,
+  knockout,
```

Comment:
maybe rename to `isKnockout`. Also, without the docs, the name of this doesn't have much meaning.

Reply:
Hmm, what about `isKnockoutStyle` so it's more obvious it's a style thing?